common/djangoapps/track/middleware.py

```diff
@@ -229,7 +229,7 @@ def get_username(self, request):
 
     def get_request_ip_address(self, request):
         """Gets the IP address of the request"""
-        ip_address = get_ip(request)
+        ip_address, _ = get_client_ip(request)
```

Comment:
```suggestion
        ip_address = get_client_ip(request)[0]
```
I'm not sure why you're using a different pattern here than in the other places.  Let's keep it consistent unless there is a reason not to.